src/GregClient/GregClient.cs

```diff
@@ -85,6 +87,43 @@ public ResponseWithContentBody<T> ExecuteAndDeserializeWithContent<T>(Request m)
 
             return new ResponseWithContent<T>(response).DeserializeWithContent();
         }
+
+        private void LogResponse(IRestResponse restResp)
+        {
+            try
+            {
+                var enableDebugLogs = Environment.GetEnvironmentVariable("DEBUG_LOGS_DYNAMO_GREG");
```

Comment:
feels more idiomatic to use a config I think - 
https://github.com/DynamoDS/Dynamo/blob/master/src/DynamoCoreWpf/App.config
https://github.com/DynamoDS/Dynamo/blob/2ef67e17e77bd12067bd33cda90c9de7ef38097b/src/DynamoCoreWpf/ViewModels/Core/DynamoViewModel.cs#L519

what do you think?

I can't think of anywhere else we use environment variables in the dynamo code base.